src/Middleware/Payload.php

```diff
@@ -28,6 +33,15 @@ public function __construct(array $options = [])
     }
 
     /**
+     * If the Request object already has a parsedBody, normally Payload will skip parsing. This is not always
+     * desirable behavior. Calling this setter allows you to override this behavior.
+     */
+    public function overrideExistingParsedBody()
```

Comment:
Thanks - personally I prefer long-and-descriptive over short, but I guess that's a taste thing. We'll go for the shorter version :)

Re: the setter's method signature: I was wondering why you'd want to "un-override" instead of just not calling the override, and ended up implementing it like this. Will be happy to change it.